### PR TITLE
Visualization and stuff

### PR DESCRIPTION
**Pull request by Z30G0D**
Figure_1

Need to handle the y axes in a better way, WIP

### CODE REVIEW

File: src/train.py [hunk 1]
```diff
@@ -119,8 +119,10 @@ def main(args):
 
     # Set device
     if not torch.cuda.is_available():
+        print('CPU!!!!!!!!!!!')
```
Comment (mosheman5) [inline src/train.py:122]: lol

File: src/conf/data/defaults.yaml [hunk 2]
```diff
@@ -6,13 +6,13 @@ data:
   data_sample_rate: 44100
   train_dataset:
     _target_: data.ClassifierDataset
-    data_path: '../tests/assets/data/'
+    data_path: '../datasets/recordings_2018/'
```
Comment (mosheman5) [inline src/conf/data/defaults.yaml:9]: Do we want to change the default file? I think we should have another configuration for running the deep voice dataset.

File: src/conf/data/defaults.yaml [hunk 3]
```diff
@@ -6,13 +6,13 @@ data:
...
     mode: train
-    metadata_path: '../tests/assets/annotations/sample_annotations.csv'
+    metadata_path: '../annotations/combined_annotations_filtered_train.csv'
     augmentations_p: 0.8
     augmentations: ${_augmentations}
     preprocessors: ${_preprocessors}
-    seq_length: 1
+    seq_length: 0.3
```
Comment (mosheman5) [inline src/conf/data/defaults.yaml:15]: why?

File: src/Notebooks/recording_vIsualization.ipynb [hunk 4]
```diff
@@ -0,0 +1,374 @@
+{
```
Comment (noamzbr) [inline src/Notebooks/recording_vIsualization.ipynb:1]: Perhaps worth separating the function from the notebook?

File: src/Notebooks/recording_vIsualization.ipynb [hunk 5]
```diff
@@ -0,0 +1,374 @@
...
+ "cells": [
```
Comment (noamzbr) [inline src/Notebooks/recording_vIsualization.ipynb:2]: Should add start_time & end_time so it won't always show all of the file.

File: src/Notebooks/recording_vIsualization.ipynb [hunk 6]
```diff
@@ -0,0 +1,374 @@
...
+  {
```
Comment (noamzbr) [inline src/Notebooks/recording_vIsualization.ipynb:3]: Add defaults to thresh, seg_length in function header

File: src/Notebooks/recording_vIsualization.ipynb [hunk 7]
```diff
@@ -0,0 +1,374 @@
...
+   "cell_type": "code",
```
Comment (noamzbr) [inline src/Notebooks/recording_vIsualization.ipynb:4]: Optional - if to many segments, separate into a few plots so plot won't be cramped

**Review by noamzbr — CHANGES_REQUESTED** [review]
Real nice! Added a few comments / small feature requests

File: src/Notebooks/recording_vIsualization.ipynb [hunk 8]
```diff
@@ -0,0 +1,374 @@
...
+   "execution_count": 159,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# idx= 10\n",
+    "# warnings.filterwarnings('ignore')\n",
+    "sample = xticks\n",
+    "%matplotlib qt\n",
```
Comment (mosheman5) [inline src/Notebooks/recording_vIsualization.ipynb:287]: should add PyQt5 to the requirements.txt file

File: src/Notebooks/recording_vIsualization.ipynb [hunk 9]
```diff
@@ -0,0 +1,374 @@
...
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Nu mber of tags to inspect: 488\n"
+     ]
+    }
+   ],
+   "source": [
+    "print('Number of seconds:', tags_to_inspect.shape[0])"
```
Comment (mosheman5) [inline src/Notebooks/recording_vIsualization.ipynb:168]: tags_to_inspect is not defined before. should be removed?

File: src/Notebooks/recording_vIsualization.ipynb [hunk 10]
```diff
@@ -0,0 +1,374 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 157,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def play_audio_plot_stft(prob, df_series, audio_path, thresh, seg_length):\n",
+    "    def get_sr(wav_path):\n",
```
Comment (mosheman5) [inline src/Notebooks/recording_vIsualization.ipynb:192]: you can use soundfile.info(wav_path).samplerate instead

File: src/Notebooks/recording_vIsualization.ipynb [hunk 11]
```diff
@@ -0,0 +1,374 @@
...
+    "        with wave.open(str(wav_path), \"rb\") as wave_file:\n",
+    "            return wave_file.getframerate()\n",
+    "        \n",
+    "    \n",
+    "    def play_audio(audio_data, sr):\n",
```
Comment (mosheman5) [inline src/Notebooks/recording_vIsualization.ipynb:197]: do we use it?

**Review by mosheman5 — CHANGES_REQUESTED** [review]
IMO, plotting all the graphs on one fig is bit "heavy" and I can't really see the details in the spectrogram.
I suggest breaking it down a plot for each graph, and to add an option to play the audio of this graph. 
If you'll use `%matplotlib notebook` instead of `%matplotlib qt` you'll have all the graphs inside the notebook, instead of a popup windows (having 10 popup windows isn't nice)

File: src/Notebooks/recording_vIsualization.ipynb (hunk 4 above)
Comment (Z30G0D) [reply src/Notebooks/recording_vIsualization.ipynb:1]: maybe not, just for convenience changing the colors easily

**Pull request merged**